a10_neutron_lbaas/vthunder/instance_manager.py

```diff
@@ -347,9 +359,6 @@ def distinct_dicts(dicts):
     return map(dict, set(hashable))
 
 
+# deprecated
 def config_instance_manager(config):
-        vth = config.get_vthunder_config()
-        ks = a10_keystone.KeystoneA10(
-            config.get('keystone_version'), config.get('keystone_auth_url'), vth)
-        imgr = InstanceManager(ks_session=ks.session)
-        return imgr
+    return InstanceManager.from_config(config)
```

Comment:
This is a minor enough change that you can nuke the deprecated method. We'll pick it up in a10-openstack.